Makefile

```diff
@@ -75,10 +75,12 @@ config: ## generate configs
 
 ci_test: ## run tests with coverage
 	go test -json -v -count 10 -timeout 20m --coverprofile=coverage.txt --covermode=atomic ./... | tee ci_test_with_coverage.json
+	go test -json -v -count 10 -timeout 20m -tags="sequential" --coverprofile=coverage.txt --covermode=atomic ./... | tee ci_test_with_coverage.json
```

Comment:
this won't start if error occurs while executing previous line. Maybe something like this?- https://github.com/insolar/go-autotests/blob/master/Makefile#L45

Reply:
and one more thing: when running `go test -tags "mytag"` also triggers to start tests without tag in addition to tests with tag. So we get two similar runs here.

Reply:
In order to reduce clutter, I've just isolated tests explicitly using db setup. Go doesn't have a good tag system, neither regex in -run is suitable for the task, so it's better to clearly statement about test state isolation like

cleaner := InitTestDB()
	defer cleaner()

and then run it the same way in count or parallel.